Gems/Atom/RHI/DX12/Code/Source/Platform/Windows/RHI/Device_Windows.cpp

```diff
@@ -491,6 +491,9 @@ namespace AZ
             }
 
             SetDeviceRemoved();
+
+            // Assert before continuing so users have a chance to inspect the TDR before the log output gets burried under the ensuing RHI errors
+            AZ_Assert(false, "GPU device lost!");
```

Comment:
Are you able to confirm why this assert didnt trigger for you?
```
bool Device::AssertSuccess(HRESULT hr)
        {
            if (hr == DXGI_ERROR_DEVICE_REMOVED)
            {
                OnDeviceRemoved();
            }

            bool success = SUCCEEDED(hr);
            AZ_Assert(success, "HRESULT not a success %x", hr);
            return success;
        }
```

Reply:
I was not unfortunately, got lots of errors after the TDR but no asserts